spot_driver/include/spot_driver/api/default_spot_api.hpp

```diff
@@ -20,7 +20,8 @@ namespace spot_ros2 {
 class DefaultSpotApi : public SpotApi {
  public:
   explicit DefaultSpotApi(const std::string& sdk_client_name,
-                          const std::optional<std::string>& certificate = std::nullopt);
+                          const std::optional<std::string>& certificate = std::nullopt,
+                          const std::optional<int8_t> timesync_timeout = std::nullopt);
```

Comment:
```suggestion
                          const std::optional<std::chrono::milliseconds> timesync_timeout = std::nullopt);
```
chrono seems like a better library for dealing with time